docs/otel/sampling.md

```diff
@@ -0,0 +1,101 @@
+SHOULD:
+
+- use the `tracestate` OpenTelemetry T-value field (`rh`) if it is present, or
+- use the `sampling.threshold` attribute value if it is present.
+
+Otherwise, no information is available about what sampling was
+performed.
+
+### Sampling randomness
+
+When determining sampling randomness from a span, implementations
+SHOULD:
+
+- use the `tracestate` OpenTelemetry R-value field (`rv`) if it is present, or
+- use the `sampling.randomness` attribute value if it is present, or
+- use the least significant 56 bits of the W3C Trace Context TraceID, as described in the W3C Trace Context Level 2 specification.
+
+Callers SHOULD NOT require trace flags to have the Trace Context Level
+2 Random flag set, in case the Trace ID is used as the source of
+randomness, because it is not reliable information at this time.
+Additionally, the W3C Trace Context Level 2 specification was based on
+the widespread use of at least 56 bits of randomness.
+
+In case the user knowingly creates TraceIDs that do not conform to the
+W3C Trace Context Level 2 specification and they wish to perform
+sampling, they SHOULD synthesize a random R-value and store it in the
+`tracestate` (Spans) or the `sampling.randomness` (Log Records)
+attribute value.
+
+## Span attributes
+
+The following attributes are recognized for Spans.  Since probability
+sampling is tightly coordinated through Context propagation and
+reflected in the W3C Trace Context `tracestate`, which is included in
+the Span data model, it is not common to express sampling randomness
+and threshold using attributes.
+
+<!-- semconv sampling.common(full,tag=otel-span-attributes) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `sampling.priority` | int | Enables prioritizing and de-prioritizing collection of telemetry. [1] | `1`; `0` | Recommended |
+
+**[1]:** This attribute is useful to express the user's desire to collect telemetry, despite the sampling configuration in place.  Non-zero values MAY prioritize the item for collection independent of probability sampling.  Zero values have the reverse effect and MAY cause telemetry not to be collected.  This convention derives from a conventional tag with the same name in the OpenTracing specification.
+<!-- endsemconv -->
+
+## Log Record attributes
+
+The following attributes are recognized for Log Records.
+
+<!-- semconv sampling.common(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `sampling.priority` | int | Enables prioritizing and de-prioritizing collection of telemetry. [1] | `1`; `0` | Recommended |
+| `sampling.randomness` | string | The source of randomness for making probability sampling decisions, when it is not otherwise recorded. [2] | `ce929d0e0e4736` | Recommended |
+| `sampling.threshold` | string | Sampling probability as specified by OpenTelemetry. [3] | `c`; `ff8` | Recommended |
```

Comment:
Is it a static value? Do we really want to have this attribute on every log record if it is or maybe it should be a resource or instrumentation scope attribute?

Reply:
It is very much not a static value and can vary with each log record, so cannot be a higher-level attribute.

Reply:
Moreover, as I've now written in the text, we can't in today's conventions safely apply a resource-level or scope-level attribute for Threshold without potentially overcounting due to spans that were sampled because of `sampling.priority`. In earlier (experimental) specifications, we had a T-value expression equivalent to "zero count", which might be a solution. For now, the text says we aren't defining these attributes for resources or scopes.